EliHaugu/liltest
# Implement docker

- Implement support for running app in Docker containers
- To get the app to work both with and without Docker, I made all calls to backend refer to a API url const set in the env file
- Improved some redundant url references in frontend (defined base url for api instead of repeating for every function)
  - However, I did not change the ones that did already have base urls but that could be improved further ie. base url not including /nodes/ when the whole file uses that endpoint anyways, because I didn't want to change too much frontend in a backend PR
- Commented out problematic call to device connection end point that is not implemented yet / for which the implementation will change

To test this PR:
- Set up app in docker by opening docker desktop (or starting docker via a terminal) and run "docker-compose up --build" in the root of the project
- Some functionality will not work if you open the frontend bc it is yet to be fixes by the frontend team, but you should be able to create and run a test flow and open the logs
